docs/intro.md

```diff
@@ -103,12 +103,12 @@ The following table lists the key participants and their roles in the UID2 [work
+| :--- |:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Workflow for DSPs](overviews/overview-dsps.md#workflow-for-dsps) (Buy-Side) | DSPs who transact on UID2 tokens in the bid stream.                                                                                                                                                                                                                                                                                                                 | [DSP Integration Guide](guides/dsp-guide.md)                                                                                                                                                                                                   |
+| [Workflow for Advertisers](overviews/overview-advertisers.md#workflow-for-advertisers) and [Workflow for Data Providers](overviews/overview-data-providers.md#workflow-for-data-providers) | Organizations that collect user data and push it to DSPs.                                                                                                                                                                                                                                                                                                           | [Advertiser/Data Provider Integration Guide](guides/advertiser-dataprovider-guide.md)                                                                                                                                                          |
+| [Workflow for Publishers](overviews/overview-publishers.md#workflow-for-publishers) | Organizations that propagate UID2 tokens to the bid stream via SSPs.<br/> NOTE: Publishers can choose to leverage the UID2 SDK for JavaScript (see [JavaScript Express Integration Guide](publisher-client-side.md) or [UID2 SDK for JavaScript Reference Guide](sdks/client-side-identity.md)) or complete their own server-only integration without using an SDK. | [JavaScript Express Integration Guide](publisher-client-side.md)<br/>[JavaScript Standard Integration Guide](guides/integration-javascript-standard.md)<br/>[Publisher Integration Guide, Server-Only](guides/custom-publisher-integration.md) |
```

Comment:
@genwhittTTD why doesn't this Workflow For Publishers section include Prebid.js guides? Is it intentional?

Reply:
Hi @sunnywu good spot. That was an omission. That home page has had very few updates. We should probably revisit the whole thing. Anyway I added Prebid. Took the links out of col 2 since they are in col 3. Though, I'm thinking that col 3 should probably link to the resources section of the publisher overview page. Trying to stick to the job at hand though.